### PR TITLE
Locks the warden's krav maga gloves to security-related areas.

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
Warden's krav maga gloves are currently the ultimate powergamer item, because with them you can instantly, undisarmably melee stun somebody and then spam quickchoke on them to kill them with essentially no counterplay. Krav maga is intended to be used for unruly prisoners, not to run out of the brig, gamer stun somebody, and then KO them in ~20 seconds with quickchoke spam. This should help with wardens running out of the brig to arrest people, because wardens won't want to lose their martial art. The range that krav maga retains is very generous, including all of the brig and prison wing, the HoS's office, the courtroom, and the gulag. This should also help warden players since the krav maga gloves will be a much less desirable powergamer item, so it won't get stolen from the warden's locker in the case of latejoiner wardens.

# Changelog

:cl:  
tweak: warden's krav maga gloves are now restricted to security related areas
/:cl:

### CODE REVIEW

**Comment by redmoogle** [comment]
what do you mean I cant beat the living hell out of my soon to be prisoners?!!

**Comment by TheRyeGuyWhoWillNowDie** [comment]
why did turdis break

File: code/datums/martial/krav_maga.dm [hunk 1]
```diff
@@ -181,14 +181,48 @@
...
 	resistance_flags = NONE
+	var/equipper = null //who's wearing the gloves?
+	var/equipped = FALSE //does the user currently have the martial art? 
+	var/list/enabled_areas = list(/area/security, 
+					/area/ai_monitored/security,
+					/area/mine/laborcamp,
+					/area/shuttle/labor,
+					/area/crew_quarters/heads/hos) //where can we use krav maga?
+
+/obj/item/clothing/gloves/krav_maga/sec/equipped(mob/user, slot)
```
Comment (redmoogle) [inline code/datums/martial/krav_maga.dm]:
```suggestion
/obj/item/clothing/gloves/krav_maga/sec/equipped(mob/user, slot)
	. = ..()
```

File: code/datums/martial/krav_maga.dm [hunk 2]
```diff
@@ -181,14 +181,48 @@
...
+	if(slot == SLOT_GLOVES)
+		equipper = user
+		START_PROCESSING(SSobj, src)
+
+/obj/item/clothing/gloves/krav_maga/sec/dropped(mob/user, slot)
```
Comment (redmoogle) [inline code/datums/martial/krav_maga.dm]:
```suggestion
/obj/item/clothing/gloves/krav_maga/sec/dropped(mob/user, slot)
	. = ..()
```

**Review by redmoogle — CHANGES_REQUESTED** [review]
This is why

**Comment by TheRyeGuyWhoWillNowDie** [comment]
holy jesus how did i get merge conflicts in the fucking changelog

**Comment by boodaliboo** [comment]
> holy jesus how did i get merge conflicts in the fucking changelog

emag to allow its old feature back

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> > holy jesus how did i get merge conflicts in the fucking changelog
> 
> emag to allow its old feature back

how about you buy a martial art instead (traitor) or use the weapons that you spawn with the ability to craft (BB) instead of relying on instastun crutches

**Comment by ghost** [comment]
Blessed PR

**Comment by Skrem7** [comment]
Will combat gloves plus still function with this PR

**Comment by ghost** [comment]
> Will combat gloves plus still function with this PR

From what I saw in the code - yes

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> Will combat gloves plus still function with this PR

Combat Gloves Plus and any other method of obtaining krav maga are not affected. ONLY the warden's krav maga gloves.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**